File: touchevent/src/main/java/com/example/touchevent/MainActivity.java
package com.example.touchevent;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.MotionEvent;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setListener(R.id.Button1);
        setListener(R.id.Button2);
        setListener(R.id.Button3);
        setListener(R.id.Button4);
        setListener(R.id.Button5);
    }

    private void setListener(int ids) {
        findViewById(ids).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                CharSequence text = ((TextView) v).getText();
                L.m3("==============",text);
            }
        });
    }

    @Override
    public boolean dispatchTouchEvent(MotionEvent ev) {
        return super.dispatchTouchEvent(ev);
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        return super.onTouchEvent(event);
    }
}

File: projects_/GradleDownloadPlugin/nomral/src/main/java/com/joyy/nomral/MainActivity.kt
package com.joyy.nomral

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        findViewById<TextView>(R.id.tv).apply {
            append(getString(R.string.batch_live_uncoming))
        }
    }
}